Probesys/bileto
# imp: Adapt the receipt email to the ticket type

## Related issue(s)



N/A

## How to test manually



1. Create an incident ticket
2. Create a request ticket
3. Verify that each receipt email is adapted to the type of the created ticket

## Checklist
